src/integration/pages/frontPage/canonicalTests.js

```diff
@@ -1,14 +1,24 @@
+import runCrossPlatformTests from './crossPlatformTests';
 import {
   runCoreCanonicalTests,
   runCanonicalAnalyticsTests,
 } from '../../common';
 
 export default () => {
+  runCrossPlatformTests();
   runCoreCanonicalTests();
   runCanonicalAnalyticsTests();
 
   it('I can see a radio schedule component with an id', () => {
-    const hasRadioSchedule = service === 'arabic';
+    const hasRadioSchedule = [
```

Comment:
I wonder if there is another way to determine if a service has a radio schedule e.g. toggle / service config. It just means that if the schedule is then added to a new service, then these tests won't run unless we remember to update the list.